### ISSUE TITLE
metric areas greater than total DGO area in scrape

### ISSUE THREAD

**Issue by jtgilbert** [Riverscapes/riverscapes-tools#1024]
results from `select * from vw_metrics where active_area > dgo_area_acres`

![area_issue](https://github.com/user-attachments/assets/0cb63c62-ed1b-4af7-a621-53e6b3c558e3)

**Comment by jtgilbert** [comment]
694 rows returned

**Comment by philipbaileynar** [comment]
The source of the problem is that there are multiple polygons for some DGOs. When we join RCAT `DGOAttributes` to RME DGOs we have to do it based on `level_path` and `seg_distance` because the FIDs can be different between RCAT and RME. But this causes duplicate data when there are multiple DGOs in RCAT with the same level_path and seg_distance. The root cause of the problem is VBET, when it creates weird centrelines and DGO polygons. These cascade to RCAT and all other downstream models.

Example: HUC [0808010105](https://data.riverscapes.net/p/6996a0be-b894-4f7a-a66a-68ae5e710430/) in Louisiana. Level path `20000200000091` and seg_distance `50` have two DGO records. They are discontinuous polygons, each with their own VBET metrics shown below.

<img width="673" alt="Screenshot 2024-09-24 at 4 23 10 PM" src="https://github.com/user-attachments/assets/66f3e0e5-3ecb-4276-b4d5-c5758ea5da69">

Bottom line. We need to review VBET in these new landscapes, such as the flat, bayou. Here's a satellite and VBET centerlines.

![Screenshot 2024-09-24 at 4 24 13 PM](https://github.com/user-attachments/assets/7ad88b90-1eed-4268-8a10-6cf252ba2b15)

**Comment by philipbaileynar** [comment]
and here's a video explaining  the problem further

https://youtu.be/PeN3z8UAG8Y
